### PR TITLE
add dimensions model

### PR DESCRIPTION
**Pull request by allandegnan**
### What

Updated the model for population type / dimensions

### How to review
Used here https://github.com/ONSdigital/dp-frontend-search-controller/pull/206/files

### Who can review

!me

@lindenmckenzie @nshumoogum

### CODE REVIEW

File: models/elasticsearch.go [hunk 1]
```diff
@@ -82,6 +82,8 @@ type ESSourceDocument struct {
 	Published       bool                `json:"published,omitempty"`
 	Language        string              `json:"language,omitempty"`
 	Survey          string              `json:"survey,omitempty"`
+	PopulationType  string              `json:"population_type,omitempty"`
```
Comment (DavidSubiros) [inline models/elasticsearch.go:85]: Could we change this to a type like the one defined here, please?
https://github.com/ONSdigital/dp-search-data-importer/blob/d5d831b47f074f7461f80f070b2976e3e6cb3be6/models/elastic.go#L63

File: transformer/transformer_test.go [hunk 2]
```diff
@@ -23,7 +23,7 @@ func TestTransform(t *testing.T) {
 			DatasetID: "",
 		},
```
Comment (DavidSubiros) [inline transformer/transformer_test.go:24]: I would suggest testing that the new transform logic works as expected. this Item model can be extended with :

```
		PopulationType: "PopLbl1",
		Dimensions: []models.ESDimensions{
			{Name: "Dim1", Label: "Lbl1", RawLabel: "RawLbl1"},
		},
```

And `esDocument1` in `prepareESMockResponse` func could be extended with:
```
		PopulationType:  models.ESPopulationType{Name: "Pop1", Label: "PopLbl1"},
		Dimensions: []models.ESDimensions{
			{Name: "Dim1", Label: "Lbl1", RawLabel: "RawLbl1"},
		},
```

File: transformer/transformer_test.go [hunk 3]
```diff
@@ -23,7 +23,7 @@ func TestTransform(t *testing.T) {
...
 	}
-	expectedESDocument2 := models.ESSourceDocument{
+	expectedItem2 := models.Item{
 		DataType:        "anyDataType2",
 		CDID:            "",
 		DatasetID:       "",
```
Comment (DavidSubiros) [inline transformer/transformer_test.go:29]: I would suggest testing that the new transform logic works as expected. This item model can be extended with:
```
		PopulationType: "PopLbl2",
		Dimensions: []models.ESDimensions{
			{Name: "Dim2", Label: "Lbl2", RawLabel: "RawLbl2"},
		},
```

And `esDocument2` in `PrepareESMockResponse` func could be extended with:
```
		PopulationType:  models.ESPopulationType{Name: "Pop2", Label: "PopLbl2"},
		Dimensions: []models.ESDimensions{
			{Name: "Dim2", Label: "Lbl2", RawLabel: "RawLbl2"},
		},
```

**Review by DavidSubiros — COMMENTED** [review]
Just a suggestion to extend the transformer unit test to validate the added fields

File: transformer/transformer_test.go (hunk 3 above)
Comment (allandegnan) [reply transformer/transformer_test.go:33]: done

**Review by DavidSubiros — APPROVED** [review]
LGTM

**Pull request merged**